prime-angular-frontend/src/app/modules/adjudication/shared/components/health-authority-table/health-authority-table.component.html

```diff
@@ -69,6 +69,14 @@
             *matCellDef="let row;"> {{ row.pec | default }} </td>
       </ng-container>
 
+      <ng-container matColumnDef="careType">
+        <th mat-header-cell
+            *matHeaderCellDef
+            scope="col"> Care Type </th>
+        <td mat-cell
+            *matCellDef="let row;"> {{ row.healthAuthorityCareType.careType | default }} </td>
```

Comment:
`HealthAuthorityCareTypeId` in table `HealthAuthoritySite` is nullable so optional chaining to be safer 
 by `healthAuthorityCareType?.careType`

Reply:
Yeah i think if a user half completes a site it will only show the data it has, will add this in